pkgs/by-name/ho/hoarder/patches/dont-lock-pnpm-version.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/package.json b/package.json
```

Comment:
Please provide comments why these patches are needed and where they are comming from.

This makes it much easier for the long term maintenance.

Reply:
I've added descriptions to all patches